### PR TITLE
feat: ZodiosOptions.rawValues

### PR DESCRIPTION
**Pull request by QzCurious**
I think it is good to separate validation and parsing logic.
I make a purposal to allow using raw request/response data like what [react-hook-form](https://github.com/react-hook-form/resolvers#api) did.

If this idea is reasonable, I'll try to finish up testing and docs.

### CODE REVIEW

File: src/zodios.types.ts [hunk 1]
```diff
@@ -567,6 +567,10 @@ export type ZodiosOptions = {
    * Should zodios validate parameters and response? Default: true
    */
   validate?: boolean | "request" | "response" | "all" | "none";
+  /**
+   * Use raw values from request/response rather than parsed values. Default: true
+   */
+  rawValues?: boolean | "request" | "response" | "all" | "none";
```
Comment (ecyrbe) [inline src/zodios.types.ts:573]: all these options don't make real sense when combined with validate, i propose to simplify with the following:
```suggestion
  /**
   * Should zodios transform the request and response ? Default: true
   */
  transform?: boolean;
```

**Review by ecyrbe — CHANGES_REQUESTED** [review]
Thank you for this PR, i agree this can be valuable.
Just one thing, i propose to rename this option to 'transform' and keep it true by default, else it would be a breaking change.

File: src/zodios.types.ts (hunk 1 above)
Comment (QzCurious) [reply src/zodios.types.ts:573]: I'd like to make it able to choose whether to transform `"response"`, `"request"` respectively? Would you mind it to be?

And it surely can be simplify to `"request" | "response" | boolean`.

**Comment by QzCurious** [comment]
Maybe I'm too rush to make a PR for it. You could just tell me to rename it without marking it all the places. Sorry for making you did that.

File: src/zodios.types.ts (hunk 1 above)
Comment (ecyrbe) [reply src/zodios.types.ts:573]: yes, go ahead.

**Comment by QzCurious** [comment]
To write test case for `ZodiosOptions.transform`, I might break current sturcture a lot. I need suggestion.

**Comment by QzCurious** [comment]
So I've decided to make test footprint as small as possible. I thought it would be easier to back to these tests once bad thing happens.

**Comment by ecyrbe** [comment]
Thank you very much.
LGTM

**Pull request merged**